styles.css

```diff
@@ -552,3 +631,189 @@ button:active {
 ::placeholder {
   color: #979493;
 }
+
+@media only screen and (min-width: 768px) {
+  .menu-icon {
+    display: none;
+  }
+
+  .nav-bar-pages {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    display: flex;
+    color: #fff;
+    text-decoration: none;
+    justify-self: right;
+    margin: 15px;
+  }
+
+  .headline {
+    flex-direction: row-reverse;
+    align-items: center;
+    padding: 200px 10px 130px 10px;
+  }
+
+  .social-media-icons {
+    flex-direction: column;
+    margin-right: 0;
+  }
+
+  footer .social-media-icons {
+    flex-direction: row;
+    margin-right: 0;
+  }
+
+  .headline-info {
+    align-items: flex-start;
+  }
+
+  .description {
+    width: 45%;
+  }
+
+  .background-img1 {
+    background-image: url('media/illustration-heading.png');
+    background-repeat: no-repeat;
+    position: absolute;
+    width: 430.89px;
+    height: 365.03px;
+    left: 65%;
+    top: 100px;
+    transform: scale(1.8);
+  }
+
+  .work1 {
+    display: grid;
+    grid-template-columns: 1.5fr 1fr;
+    grid-template-rows: 1fr;
+    width: 90%;
+    height: auto;
+    justify-items: left;
+    align-items: flex-start;
+    max-width: 1100px;
+  }
+
+  .recent-work-container {
+    flex-direction: row;
+    align-items: center;
+  }
+
+  .vector-line1 {
+    margin-top: 7px;
+  }
+
+  .multi-post-image {
+    grid-column: 1/2;
+    grid-row: 1/2;
+    height: 100%;
+    max-height: 600px;
+    align-self: center;
+  }
+
+  .work1-info {
+    display: flex;
+    flex-direction: column;
+    margin-left: 20px;
+    height: auto;
+  }
+
+  .work1 h3 {
+    margin: 0 0 12px 0;
+  }
+
+  .work1 p {
+    width: auto;
+    height: auto;
+  }
+
+  .wrapper {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+    width: auto;
+    height: auto;
+    gap: 30px;
+    justify-content: center;
+    transform: scale(auto);
+  }
+
+  .project1 {
+    grid-column: 1/2;
+    grid-row: 1/2;
+  }
+
+  .project2 {
+    grid-column: 2/3;
+    grid-row: 1/2;
+  }
+
+  .project3 {
+    grid-column: 3/4;
+    grid-row: 1/2;
+  }
+
+  .project4 {
+    grid-column: 1/2;
+    grid-row: 2/3;
+  }
+
+  .project5 {
+    grid-column: 2/3;
+    grid-row: 2/3;
+  }
+
+  .project6 {
+    grid-column: 3/4;
+    grid-row: 2/3;
+  }
```

Comment:
I suggest that you avoid styling each card individually, this helps to avoid repeating yourself (DRY). The class that they have in common could be styled instead.